temporal-testing/build.gradle

```diff
@@ -53,6 +55,23 @@ task testServiceServer(type: CreateStartScripts) {
     dependsOn 'sourcesJar'
 }
 
+graal {
```

Comment:
temporal-testing now is a mix of the testing server with some java-specific unit test helpers.
Do you think it's time to move test server into temporal-test-server and make temporal-testing depend on it?
So all this graal config can be in a more appropriate separate module than temporal-testing.

Reply:
I would do it but doesn't have to be in this PR